configs/neox_arguments.md

```diff
@@ -1307,6 +1445,14 @@ Text Generation arguments
 
 
 
+- **eval_task_limit**: int
```

Comment:
This is the only new argument in this PR. The updates elsewhere to this file are from running `configs/gen_docs.py`.